File: examples/hello_world/hello_world.js
(function() {
  var app;
  app = molly(function() {
    this.route('/examples/hello_world/', function() {
      return alert('hello');
    });
    this.route('/examples/hello_world/users/', function() {
      return alert('Viewing all users');
    });
    return this.route('/examples/hello_world/users/:id', function(id) {
      return alert("Viewing user: " + id);
    });
  });
}).call(this);

File: examples/hello_world/hello_world.cs
app = molly () ->

    @route '/examples/hello_world/', () ->
        alert 'hello'

    @route '/examples/hello_world/users/', () ->
        alert 'Viewing all users'

    @route '/examples/hello_world/users/:id', (id) ->
        alert "Viewing user: #{ id }"
